src/base64.h

```diff
@@ -3,7 +3,8 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "util.h"
+#include "util.h"  // NOLINT(build/include_inline)
```

Comment:
Question: is there a way to avoid this?

Reply:
Yes – the “right” way to do this would be to move the implementation of `base64_decode_fast` (and possibly other functions) to a `base64-inl.h` file. That file could then include `util-inl.h`, because it is an implementation header as well.